src/main/java/org/mule/service/http/impl/service/client/GrizzlyHttpClient.java

```diff
@@ -421,6 +424,18 @@ protected Request createGrizzlyRequest(HttpRequest request, HttpRequestOptions o
     return reqBuilder.build();
   }
 
+  private void setStreamingBodyToRequestBuilder(HttpRequest request, RequestBuilder builder) throws IOException {
+    if (StreamingUtils.isRequestStreamingEnabled()) {
```

Comment:
static import

Reply:
It is now resolved inside the same class